website_and_docs/content/documentation/webdriver/troubleshooting/logging.en.md

```diff
@@ -9,70 +9,165 @@ description: >
+is <i>`:warn`</i>.</p>
 
-**WARN**
+<p>To change the level of the logger:</p>
+<pre>
+  Selenium::WebDriver.logger.level = :fatal
+</pre>
+<br>
+<p><b>WARN</b></p>
 
-Warnings include everything we want users to be aware of by default. This is mostly used
+<p>Warnings include everything we want users to be aware of by default. This is mostly used
 for deprecations. For various reasons, Selenium project does not follow standard Semantic Versioning practices.
 Our policy is to mark things as deprecated for 3 releases and then remove them.
 As such, Ruby logs deprecations as warnings, specifying what is changing, what needs to be
 used instead. It may include additional messages, and always includes an ID.
+</p>
 
-For example:
+<p>For example:</p>
+<pre>
 > 2022-12-24 16:07:09 WARN Selenium [DEPRECATION] [:jwp_caps] \`Capabilities#version=\` is deprecated. Use \`Capabilities#browser_version=\` instead.
-
-Because these items can get annoying, we've provided an easy way to turn them off.
-
-To turn off a specific warning, set the ID to ignore:
-```ruby
-Selenium::WebDriver.logger.ignore(:jwp_caps)
-```
-It accepts an `Array` to turn off multiple IDs:
-```ruby
-Selenium::WebDriver.logger.ignore(%i[jwp_caps pause pauses])
-```
-To turn off all *deprecation* notices:
-```ruby
-Selenium::WebDriver.logger.ignore(:deprecations)
-```
-
-**INFO**
-
-This is where the most useful information gets logged. Selenium logs the endpoints and payloads
+</pre>
+<br>
+<p>Because these items can get annoying, we've provided an easy way to turn them off.</p>
+
+<p>To turn off a specific warning, set the ID to ignore:</p>
+<pre>
+  Selenium::WebDriver.logger.ignore(:jwp_caps)
+</pre>
+<br>
+<p>It accepts an `Array` to turn off multiple IDs:</p>
+<pre>
+  Selenium::WebDriver.logger.ignore(%i[jwp_caps pause pauses])
+</pre>
+<br>
+<p>To turn off all <i>deprecation</i> notices:</p>
+<pre>
+  Selenium::WebDriver.logger.ignore(:deprecations)
+</pre>
+<br>
+<p><b>INFO</b></p>
+
+<p>This is where the most useful information gets logged. Selenium logs the endpoints and payloads
 sent to and received from the driver or server. This is a great way to see what Selenium is actually
 doing under the hood, and can be used to determine if it is Selenium code or driver code that
 is causing a problem. (Unfortunately, we can't blame the driver if Selenium is sending incorrect syntax).
+</p>
+
+<p><b>DEBUG</b></p>
+
+<p>This is less useful information where we log things about the servers and the sockets, and header information, etc.
+Debug mode is set if either <i>`$DEBUG`</i> is true or <i>`ENV['DEBUG']`</i> has a value.</p>
+{{< /tab >}}
+
+{{< tab header="JavaScript" >}}
+<p>JS has implemented logging APIs. It has also added LogManager to maintain a collection of loggers.</p>
+<p><b>1. Getting a logger can be done in two ways:</b></p>
+<pre>
+  const logging = require('selenium-webdriver/lib/logging')
+</pre>
+<br>
+<p>First method using manager</p>
```

Comment:
Is the only reason to use the manager if you need more than one logger?

Reply:
I looked further into LogManager. Though it helps us to store loggers it is not required in our case. We don't need to explicitly use a LogManager. Selenium maintains a LogManager when it starts. I have cleaned the approach in the new commit.